jade/cli/submit_jobs.py

```diff
@@ -150,6 +160,16 @@ def submit_jobs(
             sys.exit(1)
         hpc_config = HpcConfig(**load_data(hpc_config))
 
+    if time_based_batching and per_node_batch_size != DEFAULTS["per_node_batch_size"]:
+        # This doesn't catch the case where the user passes --per-node-batch-size=default, but
+        # I don't see that click provides a way to detect that condition.
```

Comment:
Not sure if it worths the effort, another idea with `click` is using `User Input Prompts` for this feature, and ensure user's inputs with interactive session.

Reply:
Good suggestion, but I will defer. We could probably make lots of changes like that.